crates/matrix-sdk-crypto/src/machine/tests/mod.rs

```diff
@@ -1322,8 +1322,8 @@ async fn test_unsigned_decryption() {
 
     // Encrypt a second message, an edit.
     let second_message_text = "This is the ~~original~~ edited message";
-    let second_message_content = RoomMessageEventContent::text_plain(second_message_text)
-        .make_replacement(first_message, None);
+    let second_message_content =
+        RoomMessageEventContent::text_plain(second_message_text).make_replacement(first_message);
```

Comment:
This change is due to https://github.com/ruma/ruma/commit/cae74897e6992833ac148ee41b08400a062f0820